addons/binding/org.openhab.binding.exec/ESH-INF/thing/thing-types.xml

```diff
@@ -29,7 +29,7 @@
             <parameter name="interval" type="integer"  required="false">
                 <label>Interval</label>
                 <description>Interval, in seconds, the command will be repeatedly executed</description>
-                <default>60</default>
+                <default>0</default>
```

Comment:
shouldn't this line be removed at all (meaning interval will simply stay unset/null)?

Reply:
@kgoderis Short ping - would you agree that this line should be removed? And if so, can you tell whether the code needs to be adapted to handle a null interval correctly?

Reply:
Do you think we can finally merge this PR without @kgoderis? All we have to do is to find a proper solution for these two lines of code:

https://github.com/openhab/openhab2-addons/blob/38abc62fe6811e6b98a2f58f22705247192e8dca/addons/binding/org.openhab.binding.exec/src/main/java/org/openhab/binding/exec/handler/ExecHandler.java#L102-L103

@pfink @kaikreuzer wdyt?